src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
```

Comment:
The `src` attribute is not set correctly. It should point to the URL of the font file (usually .woff, .woff2, .ttf, etc.), not to the Google Fonts specimen page. You can use the Google Fonts API to get the correct URL. Also, it's a good practice to specify `font-display: swap;` to ensure text remains visible during webfont loading.